news/news-4.0.0.md

```diff
@@ -0,0 +1,165 @@
+* Improved handling of webseed servers that do not support [Range](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Range) requests or [206 Partial Content](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/206). ([#4601](https://github.com/transmission/transmission/pull/4601))
+* Improved handling of webseed servers that return a content-encoding that was not requested. ([#4609](https://github.com/transmission/transmission/pull/4609))
+* Fixed `4.0.0-beta.3` potential socket leak. ([#4616](https://github.com/transmission/transmission/pull/4616))
+* Fixed `4.0.0-beta.3` regression that failed to detect largefile build flags on 32bit systems. ([#4627](https://github.com/transmission/transmission/pull/4627))
+* Fixed `4.0.0-beta.1` UI bug when removing a tracker from the announce list. ([#4635](https://github.com/transmission/transmission/pull/4635))
+* Improved error handling when receiving corrupt piece data from peers. ([#4665](https://github.com/transmission/transmission/pull/4665))
+* Fixed `4.0.0-beta.1` FTBFS error on CentOS 7. ([#4673](https://github.com/transmission/transmission/pull/4673), [#4675](https://github.com/transmission/transmission/pull/4675))
+* Made small performance improvements in libtransmission. ([#4577](https://github.com/transmission/transmission/pull/4577), [#4679](https://github.com/transmission/transmission/pull/4679))
+* Dropped [libiconv](https://www.gnu.org/software/libiconv/) dependency in libtransmission. ([#4565](https://github.com/transmission/transmission/pull/4565))
+* Updated 403 RPC error message. ([#4567](https://github.com/transmission/transmission/pull/4567))
+* Bumped [libdeflate](https://github.com/ebiggers/libdeflate) snapshot to 1.17. ([#4596](https://github.com/transmission/transmission/pull/4596))
+
+### macOS Client
+
+* Fixed two Help menus in macOS. ([#4500](https://github.com/transmission/transmission/pull/4500))
+* Fixed bug that caused magnet links to always be paused when added. ([#4528](https://github.com/transmission/transmission/pull/4528))
+* Fixed `4.0.0-beta.1` regression that broke the Piece View "blinking" when a piece completes. ([#4587](https://github.com/transmission/transmission/pull/4587))
+* Ensured that the preferences window is centered the first time it is shown. ([#4659](https://github.com/transmission/transmission/pull/4659))
+* Fixed `4.0.0-beta.1` regression that incorrectly handled corrupt blocklists. ([#4705](https://github.com/transmission/transmission/pull/4705))
+* Increased the font size in the main window's torrent list. ([#4557](https://github.com/transmission/transmission/pull/4557))
+* Fixed [libpsl](https://github.com/rockdaboot/libpsl) dependency build issue on macOS. ([#4642](https://github.com/transmission/transmission/pull/4642))
+
+### Qt Client
+
+* Made display order of  speed limits consistent between Properties, Details dialogs. ([#4677](https://github.com/transmission/transmission/pull/4677))
```

Comment:
Typo in line 78: "of speed" <- no need for double space between these 2 words.

Reply:
Fixed c5ded48fec22fc581c06f227e8396abf76fd6420